capability/capability_test.go

```diff
@@ -21,6 +21,16 @@ const (
 	maxLastCap = CAP_CHECKPOINT_RESTORE
 )
 
+func requirePCapSet(t *testing.T) {
+	pid, err := NewPid(0)
```

Comment:
nit: use `NewPid2` and `Load` instead. `NewPid` is deprecated and we should not use it even in our own code, otherwise we're giving a bad example.